client/src/locales/en.json

```diff
@@ -152,7 +152,9 @@
   "list.index.hide-members": "Hide sacks members",
   "list.index.show-members": "Show sacks members",
   "list.index.archivization": "'{name}' sack archivization...",
+  "list.index.changing-type": "Changing '{name}' sack's type  to {value}...",
   "list.index.question": "Do you really want to archive the '{name}' sack?",
+  "list.index.change-type-question": "Do you really want to change the '{name}' sack to {value}?",
```

Comment:
I don't like those `Do you really...` questions. Maybe 'Are you sure you want to...' would sound better?